src/obfl-specification.html

```diff
@@ -1557,18 +1562,121 @@ <h4 id="L735" class="include">table-of-contents</h4>
       </dl>
     </dd>
   <dt>Content Model</dt>
-    <dd>toc-entry [1 or more]</dd>
+    <dd>toc-block [1 or more]</dd>
+</dl>
+</div>
+
+<h4 id="L735" class="include">toc-block</h4>
+
+<p>The toc-block element defines a block of text that can be used within a
+table of contents. It is an extension of the block element with toc-entries.</p>
+
+<div class="sidebar markup">
+<dl>
+  <dt>Usage</dt>
+    <dd>table-of-contents, toc-block</dd>
+  <dt>General Attributes</dt>
+  <dd><p>This element supports the <a href="#block-atts">Block</a>, <a href="#margin-atts">Margin</a>
+    and <a href="#padding-atts">Padding</a> attributes groups.</p>
+      <p>This element supports the <a href="#borders">Borders</a> attribute
```

Comment:
Should the ```toc-block``` element inherit all the attributes of a block? Are border, padding, and margin needed or desirable?

Also, most of the special attributes seem to be tailored for longer text blocks instead of entry blocks.

Reply:
I think border, padding and margin are indeed needed, as well as (almost) all the other `block` attributes. `toc-entry` also supported all of these.

Reply:
Wait, I was wrong, `toc-entry` currently does not support the "Special attributes". So maybe these can be skipped for now, but I do think that most of them could be useful as well (only the ones related to volume breaking are not useful).

Reply:
Now the only allowed attibute groups are: Borders, Block, Margin, Padding.